symfony/all-my-sms-notifier/5.3/manifest.json

```diff
@@ -1,4 +1,13 @@
 {
+    "add-lines": [
+        {
+            "file": "config/packages/notifier.yaml",
+            "position": "after_target",
+            "warn_if_missing": true,
+            "target": "        texter_transports:",
+            "content": "            allmysms: '%env(ALLMYSMS_DSN)%'"
```

Comment:
I'm not sure if we will find a way to convert this from `.yaml` to `.php` files. I think this mechanism should not be used for `config/packages` files and another mechanism would be better here.

https://github.com/symfony/flex/issues/984